src/cr/cube/cube.py

```diff
@@ -289,15 +289,18 @@ def inflate(self):
         )
 
     @lazyproperty
-    def is_mr_by_itself(self):
-        """True if the cube contains MRxItself as last 2 dimensions."""
+    def is_mr_aug(self):
```

Comment:
I like the shortening of the prop name. However, coming from crunchdb after a while, I find myself _guessing_ what the `MR_AUG` cube is. I _think_ it's the one we use to calculate the selected vs non-selected significance, that we did sometime after Montreal. Anyhow, I'd feel more reassured, as a reader, if I found at least one more sentence in the last paragraph, that would give more context, as to the usage of such cubes. Something like. An `MR_AUG` cube is used to calculate statistical significance of complete-cases, and is created by artificially augmenting the original cube's dimensions, such as [MR], or [CAT x MR], by an additional MR dimension. The extra MR dimension must be the _same_ as the one originally figuring in the cube's dimensions. After the augmentation, it becomes [MR x MR], or [CAT x MR x MR].

After typing this out, I'm aware that it's a bit too elaborate, so I leave it to your best judgement :)